addons/web_editor/static/src/js/editor/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -1474,6 +1498,352 @@ describe('Paste', () => {
+                    stepFunction:  async editor => {
+                        await pasteHtml(editor, unformat(`
+                            <ul>
+                                <li>abc</li>
+                                <li>def</li>
+                                <ul>
+                                    <li>ghi</li>
+                                    <li>jkl</li>
+                                </ul>
+                                <ol>
+                                    <li>mno</li>
+                                    <li>pqr</li>
+                                </ol>
+                            </ul>
+                        `));
+                    },
+                    contentAfter: unformat(`
+                        <ul>
+                            <li>abc</li>
+                            <li>def</li>
+                            <li class="oe-nested">
+                                <ul>
+                                    <li>ghi</li>
+                                    <li>jkl</li>
+                                </ul>
+                            </li>
+                            <li class="oe-nested">
+                                <ol>
+                                    <li>mno</li>
+                                    <li>pqr[]</li>
+                                </ol>
+                            </li>
+                        </ul>
+                    `),
+                });
+            });
+            it('should convert multiple paragraphs into a checklist', async () => {
+                await testEditor(BasicEditor, {
+                    removeCheckIds: true,
+                    contentBefore: `<ul class="o_checklist"><li>[]<br></li></ul>`,
+                    stepFunction:  async editor => {
+                        await pasteHtml(editor, unformat(`
+                            <p>abc</p>
+                            <p>def</p>
+                            <p>ghi</p>
+                            <p>jkl</p>
+                            <p>mno</p>
+                        `));
+                    },
+                    contentAfter: unformat(`
+                        <ul class="o_checklist">
+                            <li>abc</li>
+                            <li>def</li>
+                            <li>ghi</li>
+                            <li>jkl</li>
+                            <li>mno[]</li>
+                        </ul>
+                    `),
+                });
+            });
+            it('should insert a list and a p tag inside a new list', async () => {
+                await testEditor(BasicEditor, {
+                    contentBefore: '<ul><li>[]<br></li></ul>',
+                    stepFunction: async editor => {
+                        await pasteHtml(editor, '<ul><li>abc</li><li>def</li></ul><p>ghi</p>');
+                    },
+                    contentAfter: '<ul><li>abc</li><li>def</li><li>ghi[]</li></ul>',
+                });
+            });
```

Comment:
This is the test you mentioned [here](https://github.com/odoo/odoo/pull/115019#discussion_r1469411170) right?

Reply:
yes